app/concepts/api/v1/users/registrations/serializer/create_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Api::V1::Users::Registrations::Serializer
+  class CreateSerializer < ApplicationSerializer
+    attributes :id, :email, :created_at
```

Comment:
Please change class & file name. Also please add:

```ruby
set_type :user
```

It will fix error with ugly type naming from file names O_o